mopidy_spotify/web.py

```diff
@@ -9,11 +9,15 @@
 import urllib
 import urlparse
 
+from contextlib import contextmanager
+from datetime import datetime
+
 import requests
 
 from mopidy_spotify import utils
 
 logger = logging.getLogger(__name__)
+INFO = logging.getLevelName('INFO')
```

Comment:
This is already available as `logging.INFO`.

Reply:
Thanks!